examples/python/tensor_transforms.ipynb

```diff
@@ -0,0 +1,611 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "kernelspec": {
+      "display_name": "Python 3",
+      "language": "python",
+      "name": "python3"
+    },
+    "language_info": {
+      "codemirror_mode": {
+        "name": "ipython",
+        "version": 3
+      },
+      "file_extension": ".py",
+      "mimetype": "text/x-python",
+      "name": "python",
+      "nbconvert_exporter": "python",
+      "pygments_lexer": "ipython3",
+      "version": "3.7.7"
+    },
+    "colab": {
+      "name": "torchvision_scriptable_transforms.ipynb",
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "accelerator": "GPU"
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "vjAC2mZnb4nz"
+      },
+      "source": [
+        "# Image transformations\n",
+        "\n",
+        "This notebook shows new features of torchvision image transformations:\n",
+        "- can accept torch tensor images as input\n",
+        "- can run on GPU\n",
+        "- can be torch jit scripted (except `Compose`, `RandomApply`, `RandomChoice`, `Lambda` transforms).\n",
+        "- can be applied on batch of tensors\n"
```

Comment:
I think it would be nice to give some more context here first.
Maybe mention that the transforms in torchvision have traditionally been PIL-centric, and as such they didn't support many things like
- multi-band images (with more than 3-4 channels)
- to be torchscripted together with your model for deployment
- support for GPU acceleration
- support for batched transformation (such as for videos)

And that now with a Tensor-first implementation (while maintaining PIL support for backwards-compatibility), we can achieve all of the above.